packages/shared/styles/components/atoms/SfBreadcrumbs.scss

```diff
@@ -1,37 +1,42 @@
 @import "../../helpers";
 
 .sf-breadcrumbs {
-  font: var(
+  @include font(
     --breadcrumbs-font,
-    font(
-      --breadcrumbs,
-      400,
-      var(--font-size-small),
-      1.6,
-      var(--body-font-family-secondary)
-    )
+    var(--font-weight-regular),
+    var(--font-size-small),
+    1.6,
+    var(--body-font-family-secondary)
   );
   &__list {
+    padding: var(--breadcrumbs-list, var(--spacer-small));
```

Comment:
we shouldn't change padding here, try to not use any outer positioning for components (like outer margins). The way they're positioned in the layout should be determined in the outer environment. :)

Reply:
I agree with Marta, also remember to add the property name to var, in this case it should be "--breadcrumbs-list-padding"

Reply:
deleted :)